README.md

```diff
@@ -98,6 +98,9 @@ Update the var.tfvars file with values specific to your environment. Following i
  * `storage_type` : Storage provisioner to configure. Supported values: nfs (For now only nfs provisioner is supported, any other value won't setup a storageclass)
  * `volume_size` : If storage_type is nfs, a volume will be created with given size in GB and attached to bastion node. Eg: 1000 for 1TB disk.
  * `volume_storage_template` : Storage template name or ID for creating the volume. Empty value will use default template.
+ * `upgrade_image` : If set, OCP cluster will be upgraded to this image version.
```

Comment:
May be a little more info here. That it is actually "openshift release image". Also, the upgrade image version should be greate than the current version and should be supported.

Reply:
Also an example command line will be very helpful for users.